backoffice/src/Models/FormModels/AddressForm.cs

```diff
@@ -0,0 +1,30 @@
+using System.ComponentModel.DataAnnotations;
+using Droits.Models.Entities;
+
+namespace Droits.Models.FormModels;
+
+public class AddressForm
+{
+    public AddressForm()
+    {
+    }
+
+    public AddressForm(Address address)
+    {
+        AddressLine1 = address.Line1;
+        AddressLine2 = address.Line2;
+        AddressTown = address.Town;
+        AddressCounty = address.County;
+        AddressPostcode = address.Postcode;
+    }
+    public string AddressLine1 { get; set; } = string.Empty;
+    public string AddressLine2 { get; set; } = string.Empty;
+    public string AddressTown { get; set; } = string.Empty;
+    public string AddressCounty { get; set; } = string.Empty;
+    [Required(ErrorMessage = "Postcode is required")]
+    [RegularExpression("([Gg][Ii][Rr] 0[Aa]{2})|((([A-Za-z][0-9]{1,2})" +
```

Comment:
This Regex looks a bit OTT. 

^(?i)([A-Z]{1,2}[0-9]{1,2}[A-Z]?)\s?([0-9][A-Z]{2})$


Should work

Reply:
Regex taken from https://github.com/stemount/gov-uk-official-postcode-regex-helper
which says it is 'Official'